JS/index.js

```diff
@@ -18,19 +17,19 @@ const ClassificationLevel = require('./models/ClassificationLevel');
         new PassengerPlane('Embraer 190', 870, 8100, 30800, 64),
         new PassengerPlane('Sukhoi Superjet 100', 870, 11500, 50500, 140),
         new PassengerPlane('Bombardier CS300', 920, 11000, 60700, 196),
-        new MilitaryPlane('B-1B Lancer', 1050, 21000, 80000, MilitaryType.TYPE_BOMBER),
-        new MilitaryPlane('B-2 Spirit', 1030, 22000, 70000, MilitaryType.TYPE_BOMBER),
-        new MilitaryPlane('B-52 Stratofortress', 1000, 20000, 80000, MilitaryType.TYPE_BOMBER),
-        new MilitaryPlane('F-15', 1500, 12000, 10000, MilitaryType.TYPE_FIGHTER),
-        new MilitaryPlane('F-22', 1550, 13000, 11000, MilitaryType.TYPE_FIGHTER),
-        new MilitaryPlane('C-130 Hercules', 650, 5000, 110000, MilitaryType.TRANSPORT),
-        new experimentalPlane("Bell X-14", 277, 482, 500, ExperimentalTypes.HIGH_ALTITUDE, ClassificationLevel.SECRET),
-        new experimentalPlane("Ryan X-13 Vertijet", 560, 307, 500, ExperimentalTypes.VTOL, ClassificationLevel.TOP_SECRET)
+        new MilitaryPlane('B-1B Lancer', 1050, 21000, 80000, militaryTypes.BOMBER),
+        new MilitaryPlane('B-2 Spirit', 1030, 22000, 70000, militaryTypes.BOMBER),
+        new MilitaryPlane('B-52 Stratofortress', 1000, 20000, 80000, militaryTypes.BOMBER),
+        new MilitaryPlane('F-15', 1500, 12000, 10000, militaryTypes.FIGHTER),
+        new MilitaryPlane('F-22', 1550, 13000, 11000, militaryTypes.FIGHTER),
+        new MilitaryPlane('C-130 Hercules', 650, 5000, 110000, militaryTypes.TRANSPORT),
+        new ExperimentalPlane("Bell X-14", 277, 482, 500, experimentalTypes.HIGH_ALTITUDE, classificationLevels.SECRET),
+        new ExperimentalPlane("Ryan X-13 Vertijet", 560, 307, 500, experimentalTypes.VTOL, classificationLevels.TOP_SECRET)
     ];
 
     let airport = new Airport(planes);
     let militaryAirport = new Airport(airport.getMilitaryPlanes());
-    let passengerAirport = new Airport(airport.getPasPl());
+    let passengerAirport = new Airport(airport.getPassengerPlanes());
     console.log(`Military airport sorted by max distance: ${Airport.print(militaryAirport.sortByMaxDistance())}`);
     console.log(`Passenger airport sorted by max speed: ${Airport.print(passengerAirport.sortByMaxSpeed())}`);
     console.log(`Plane with max passenger capacity: ${Airport.print(passengerAirport.getPassengerPlaneWithMaxPassengersCapacity())}`);
```

Comment:
It would be a nice idea to export the `airport` variable to your tests, as it has the same structure here and in the tests.